qa/workunits/rbd/rbd_groups.sh

```diff
@@ -217,23 +248,25 @@ echo "PASSED"
 echo "TEST: create remove snapshots of consistency group"
 image="test_image"
 group="test_consistency_group"
-snap="group_snap"
-new_snap="new_group_snap"
-sec_snap="group_snap2"
+snaps=("group_snap1" "group_snap2" "group_snap3" "group_snap4")
 create_image $image
 create_group $group
+create_snapshot $group ${snaps[0]}
+check_snapshot_info $group ${snaps[0]}
 add_image_to_group $image $group
-create_snapshot $group $snap
-check_snapshot_in_group $group $snap
-rename_snapshot $group $snap $new_snap
-check_snapshot_not_in_group $group $snap
-create_snapshot $group $sec_snap
-check_snapshot_in_group $group $sec_snap
-rollback_snapshot $group $new_snap
-remove_snapshot $group $new_snap
-check_snapshot_not_in_group $group $new_snap
-remove_snapshot $group $sec_snap
-check_snapshot_not_in_group $group $sec_snap
+create_snapshot $group ${snaps[1]}
+check_snapshot_in_group $group ${snaps[1]}
```

Comment:
This call seems redundant as `check_snapshot_info()` is called immediately after and would fail if the snapshot isn't there.